vars/ubuntu14.yml

```diff
@@ -1,21 +1,34 @@
+#
+# Copyright 2015 Midokura SARL
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
 ---
-deployment_name   : ubuntu14-50-liberty
-num_nsdb_nodes    : 3
-num_comp_nodes    : 3
-num_edge_nodes    : "{{ os_edges_net_id | length }}"
-admin_password    : somepassword
-apt_user          : apt_user
-apt_password      : apt_pass
-midonet_version   : '5.0'
-openstack_version : liberty
-ubuntu_version    : trusty
-os_virt_type      : qemu
-public_subnet_cidr: 200.200.19.0/24
-ssh_user          : ubuntu
-midonet_repo      : "midonet-{{ midonet_version.split(\".\")[0] }}"
-midonet_dist      : stable
-midonet_os_dist   : stable
-midonet_os_repo   : "openstack-{{openstack_version}}"
-
-# Set if deploying on OpenStack
-os_image_id       : 1ebcca58-e8c8-4dff-afa3-72aab08a87ca
+deployment_name  : ubuntu14
+num_nsdb_nodes   : 1
+num_comp_nodes   : 1
+num_edge_nodes   : "{{ os_edges_net_id | length }}"
+admin_password   : midonet
+midonet_version  : 2:5.0.2~rc1
```

Comment:
looks like `midonet_version 5.0.2` is not available.
Here is the output for `apt-cache policy midonet-cluster`:

`midonet-cluster:
  Installed: (none)
  Candidate: 2:5.1.0~rc0
  Version table:
     2:5.1.0~rc0 0
        500 http://builds.midonet.org/midonet-5/ testing/main all Packages
     2:5.0.2~rc4 0
        500 http://builds.midonet.org/midonet-5/ testing/main all Packages
     2:5.0.2~rc3 0
        500 http://builds.midonet.org/midonet-5/ testing/main all Packages
     2:5.0.2~rc2 0
        500 http://builds.midonet.org/midonet-5/ testing/main all Packages
     2:5.0.2~rc1 0
        500 http://builds.midonet.org/midonet-5/ testing/main all Packages
     2:5.0.2~rc0 0
        500 http://builds.midonet.org/midonet-5/ testing/main all Packages
`

So when I try to install `midonet-cluster`, it gives me this error:
`failed: E: Version '2:5.0.2' for 'midonet-cluster' was not found`